src/modules/claim/resolvers/credential.resolver.ts

```diff
@@ -74,9 +74,13 @@ export class RoleCredentialResolver implements CredentialResolver {
    */
   serviceEndpointsToEIP191(tokens: string[]): RoleEIP191JWT[] {
     return tokens
-      .map((token) => {
+      ?.map((token) => {
```

Comment:
I'm curious on your rationale for the optional chaining operator here. I think I might prefer to have an exception be thrown here if `tokens` was `undefined` rather than `undefined` be returned. What are your thoughts here?

Reply:
removed